person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly make sure to add getter and setter methods before initialize constructor as below:
```ruby
attr_accessor :name, :age 
  attr_reader :id 
```

Reply:
Thank you for the feedback